spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -33,10 +33,10 @@ To meet those requirements, we introduce 2 new claims to the MP-JWT:
 
 * "upn": A human readable claim that uniquely identifies the subject or user principal of the token, across
 the MicroProfile services the token will be accessed with.
-* "groups": The token subject's group memberships that will be mapped to Java EE style application
-level roles in the MicroProfile service container.
+* "groups": A list of group names to be assigned to the principal of the MP-JWT. This typically will require a mapping at the application container level to application roles. A one-to-one mapping between group names and application role names is done by default when no custom mapping is defined in an implementation specific way.
```

Comment:
@rdebusscher How does it look ? It does look clearer than the original text, I agree :-) CC @radcortez

Reply:
We should say comma separated group names.

Reply:
looks goods to me now.

Reply:
@Emily-Jiang 

> We should say comma separated group names.

This will add to the confusion IMHO. `groups` claim is an array, so adding the `comma-separated` would not work well I believe. I'm not sure where commas would apply - it is an implementation specific aspect I guess. Thanks